modules/virt-preventing-workload-updates-during-control-plane-only-update.adoc

```diff
@@ -8,6 +8,28 @@
 
 When you update from one Extended Update Support (EUS) version to the next, you must manually disable automatic workload updates to prevent {VirtProductName} from migrating or evicting workloads during the update process.
 
+[IMPORTANT]
+====
+In {product-title} 4.16, the underlying {op-system-first} upgraded to version 9.4 of {op-system-base-full}. To operate correctly, all `virt-launcher` pods in the cluster need to use the same version of {op-system-base}.
+
+After upgrading to {product-title} 4.16 from an earlier version, re-enable workload updates in {VirtProductName} to allow `virt-launcher` pods to update. Before upgrading to the next {product-title} version, verify that all VMIs use up-to-date workloads:
+
+[source,terminal]
+----
+$ oc get kv kubevirt-kubevirt-hyperconverged -o json -n openshift-cnv | jq .status.outdatedVirtualMachineInstanceWorkloads
+0
+----
+
+If the previous command returns a value larger than 0, list all VMIs with outdated `virt-launcher` pods and live migrate them to a new version manually:
```

Comment:
```suggestion
If the previous command returns a value larger than 0, list all VMIs with outdated `virt-launcher` pods and manually use live migration to migrate them to a new version:
```

I'm not sure if "live migration" should be used as a verb.

Reply:
There are 5 occurrences of this across the OpenShift docs, which is why I initially tolerated it, but you are right. I rephrased it with a proper verb.